src/components/elements/Map-mapbox/Map.tsx

```diff
@@ -257,23 +263,59 @@ export const MapContainer = ({
     }
   };
 
+  const flyToPolygonBounds = async (poly_id: string) => {
+    const bbox = await fetchGetV2TerrafundPolygonBboxUuid({ pathParams: { uuid: poly_id } });
+    const bounds: any = bbox.bbox;
+    if (!map.current) {
+      return;
+    }
+    map.current.fitBounds(bounds, {
+      padding: 100,
+      linear: false
+    });
+  };
+
+  const { mutate: putMutate } = usePutV2TerrafundPolygonUuid();
+  const { mutate: postMutate } = usePostV2GeometryUUIDNewVersion();
```

Comment:
rename this two, I suggest `updateGeometry` and `createGeometry`, but up to you, I just find very hard to understand the put and post mutate